cmd/harborAPI.go

```diff
@@ -129,6 +130,26 @@ func update(token string, url string, data interface{}) {
 	}
 }
 
+// GetLogs returns a string of all container logs for a shipment
+func GetLogs(barge string, shipment string, env string) string {
+    var url string = helmitURI + "/harbor/" + barge + "/" + shipment + "/" + env
+
+		log.Println(url)
```

Comment:
I've been checking for the --verbose flag before writing out debug/extraneous type info...

```go
if Verbose {
  log.Println("extra info")
}
```